src/theory/symmetry_detect.h

```diff
@@ -34,6 +32,30 @@ namespace CVC4 {
 class SymmetryDetect
 {
  public:
+  /**
+   * Constructor
+   * */
+  SymmetryDetect()
+  {
+    d_trueNode = NodeManager::currentNM()->mkConst<bool>(true);
+    d_falseNode = NodeManager::currentNM()->mkConst<bool>(false);
+  }
+
+  /**
+   * Destructor
+   * */
+  ~SymmetryDetect() {}
+
+  /** Get the final partition after symmetry detection */
```

Comment:
Explain the theoretical property ensured by assertions here, along the lines of "if a vector in parts contains two variables x and y, then assertions and assertions{ x -> y, y -> x } are equisatisfiable."